added failstate

Selawen opened the pull request.

Selawen committed:
added failstate

Pheraone commented:
Hoi Inger, 
Ik heb gekeken naar je pull request maar er zijn een paar dingen die ik niet helemaal begrijp en er zitten merge conflicts in.

Selawen committed:
Merge branch 'master' into Development

Selawen commented:
resolved the merge conflicts. hij zou nu moeten werken

Selawen committed:
Resolved errors created around 23:59

- changed enemy used from test to actual
- set the text in the win/death screen to the right type
- changed instances of variables that still tried to use old names
